contributing_to_docs/templates/overview_topic_template.adoc

```diff
@@ -1,8 +1,31 @@
 = Overview
 {product-author}
 {product-version}
-:data-uri:
-:icons:
+:icons: font
 :experimental:
 
---Remove this and add a brief introduction here--
+--Start with a clear introduction to the topic and what information customers can expect to find in the topic--
```

Comment:
I'll get this fixed, but remember that we're no longer using terms such as chapter or book.